### PR TITLE
Initial commit for ANF CRR in PowerShell

### PR DESCRIPTION
**Pull request by AdelGhabboun**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: src/CreateANFCRR.ps1 [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+    -Name $SecondaryNetAppVolumeName `
+    -UsageThreshold $NetAppVolumeSize `
+    -ProtocolType "NFSv4.1" `
+    -ServiceLevel $SecondaryServiceLevel `
+    -SubnetId $SecondarySubnetId `
+    -CreationToken $SecondaryNetAppVolumeName `
+    -ExportPolicy $ExportPolicy `
+    -ReplicationObject $DataReplication
+
+WaitForANFResource -ResourceType Volume -ResourceId $NewSecondaryVolume.Id -CheckForReplication $True
```
Comment (audunn) [inline src/CreateANFCRR.ps1:322]: Perhaps explain the meaning of checkForReplication and its purpose

File: src/CreateANFCRR.ps1 [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+
+Write-Verbose -Message "Azure NetApp Files Secondary Volume has been created successfully: $($NewSecondaryVolume.Id)" -Verbose
+
+#Authorize the Primary volume
+Write-Verbose -Message "Authorizing replication in source region ..." -Verbose
+Approve-AzNetAppFilesReplication -ResourceGroupName $PrimaryResourceGroupName `
```
Comment (audunn) [inline src/CreateANFCRR.ps1:328]: Perhaps also show checking for successful replication (replication status) and/or waiting for replication status (some replication operations require a specific Replication Status to be reached)

File: src/CreateANFCRR.ps1 (hunk 1 above)
Comment (AdelGhabboun) [reply src/CreateANFCRR.ps1:322]: CheckForReplication flag is intended to go deeper and check the replication status when creating the secondary Volume. I guess the name should be CheckForReplicationStatus

File: src/CreateANFCRR.ps1 (hunk 2 above)
Comment (AdelGhabboun) [reply src/CreateANFCRR.ps1:328]: Makes sense. I will add that

File: src/CreateANFCRR.ps1 (hunk 1 above)
Comment (audunn) [reply src/CreateANFCRR.ps1:322]: Makes sense, I figured it would help to explain to the audience in the code sample.

File: README.md [hunk 3]
```diff
@@ -1,57 +1,103 @@
...
+- powershell
+products:
+- azure
+- azure-netapp-files
+description: "This project demonstrates how to use PowerShell with NetApp Files SDK for Microsoft.NetApp resource provider to deploy a cross-region replication for NFS 4.1 Volume."
+---
+
+# Azure NetAppFiles Cross-Region Replication (CRR) Sample Script PowerShell 
+
+This project demonstrates how to deploy a cross-region replication with enabled with NFS 4.1 protocol volume using PowerShell and Azure NetApp Files SDK.
```
Comment (audunn) [inline README.md:13]: Perhaps a link to general documentation of the CRR feature? Something like  https://docs.microsoft.com/en-us/azure/azure-netapp-files/cross-region-replication-introduction

**Review by audunn — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,57 +1,103 @@
...
+	| Primary Capacity pool 
+		| Primary NFS v4.1 Volume 
+		
+ * Secondary ANF Account
+	| Secondary Capacity pool
+		| Secondary NFS v.1 Data Replication Volume with referencing to the primary volume Resource ID
+			
+ * Authorize Source volume with Desitnation Volume Resource ID
+ 
+ * Finally, the clean up process takes place (not enabled by default, please change the variable shouldCleanUp to true at CreateANFCRR.ps1 file if you want the clean up code to take a place),deleting all resources in the reverse order following the hierarchy otherwise we can't remove resources that have nested resources still live. You will also notice that the clean up process uses a function called WaitForNoANFResource, at this moment this is required so we can workaround a current ARM behavior of reporting that the object was deleted when in fact its deletion is still in progress.
```
Comment (audunn) [inline README.md]: Perhaps a clean up only option as a convenience, say if you run without shouldCleanUp to study the resources created etc. and then would clean up afterwards.

**Pull request merged**